apps/site/assets/ts/schedule/components/ScheduleDirection.tsx

```diff
@@ -141,13 +145,26 @@ const ScheduleDirection = ({
         />
         <ScheduleDirectionButton dispatch={dispatch} />
       </div>
-      {mapState.data && (
+      {!staticMapData && mapState.data && (
         <Map
           channel={`vehicles:${route.id}:${state.directionId}`}
           data={mapState.data}
           shapeId={shapeId}
         />
       )}
+      {staticMapData && (
+        <>
+          <img
+            src={staticMapData.img_src}
+            alt="Route Map"
```

Comment:
I did because it failed linting without an `alt` 😅 But perhaps because of that, I didn't give it the level of thought I should have. I'll make it more descriptive.